### ISSUE TITLE
https://melpa.org/packages/magit-section-20200123.2235.tar: Not found

### ISSUE THREAD

**Issue by codeasone** [alphapapa/sbuffer.el#3]
Tried installing with: 
```
(use-package sbuffer
  :quelpa (sbuffer :fetcher github :repo "alphapapa/sbuffer.el"))
```
It fails with the following messages:

```
Fetcher: github
Source: alphapapa/sbuffer.el

Cloning https://github.com/alphapapa/sbuffer.el.git to /home/mark/.config/emacs/quelpa/build/sbuffer/
Saving file /home/mark/.config/emacs/quelpa/packages/sbuffer-20200219.347.el...
Wrote /home/mark/.config/emacs/quelpa/packages/sbuffer-20200219.347.el
Wrote /home/mark/.config/emacs/quelpa/packages/sbuffer-readme.txt
Contacting host: melpa.org:443
package--with-response-buffer-1: https://melpa.org/packages/magit-section-20200123.2235.tar: Not found
```

My environment is: 

```
GNU Emacs 28.0.50 (build 1, x86_64-pc-linux-gnu, GTK+ Version 3.22.30, cairo version 1.15.10) of 2020-01-15
Magit 20200208.551, Git 2.17.1, Emacs 28.0.50, gnu/linux
```

**Comment by alphapapa** [comment]
Update your package list.

**Comment by codeasone** [comment]
Thanks for the advice @alphapapa. After refreshing I managed to get `sbuffer` installed (and it's a keeper, improved ergonomics over `ibuffer` imo) but I did hit a couple of other minor snags. 

`magit-preserve-section-visibility-cache` was reported as undefined. 

I addressed this with: 

```elisp
(use-package sbuffer
  :quelpa (sbuffer :fetcher github :repo "alphapapa/sbuffer.el")
  :after magit
  :bind (("C-x b" . sbuffer)
         ("C-x C-b" . sbuffer)
        ))
``` 
But this relied on `magit` being loaded - which I had configured with `defer: t` - meaning `M-x sbuffer` would not work until I'd completed a workflow like `magit-status`. Loading `magit` with `:demand t` in my `init.el` sorted everything at the expense of a few ms on start-up time. 

I hope this feedback is helpful. If installation can accommodate these idiosyncrasies of configuration better, that would be great.  

Thanks for your work on this package.
